rpc/common/moc_common_common.proto

```diff
@@ -103,13 +103,25 @@ message Version {
 	string number = 1;
 }
 
+enum PlacementStatusType {
+	UNKNOWN_PLACEMENT = 0;
+	ALIGNED = 1;
+	NOT_ALIGNED = 2;
+}
+
+message PlacementStatus  {
```

Comment:
Can we move the placement status to `ZoneConfiguration` ?

Reply:
Forgot to add couple of reasons for adding it in common place.
1.  Simplify status parsing for above layers 
2. VM placement status is applicable to placement group also. 

So, it will be better to keep it in common status or define a separate placement status field in VM spec. 
Please share your thoughts on this.

Reply:
okay, in that case we can define the placement status in common and reuse it wherever needed

Reply:
Make it a field for VM or hide the status field for other entities

Reply:
Made changes to hide placement status field if it is not set.

Reply:
Created two VMs one with zone associated and another without. Captured Get-MocVirtualMachine output for both. |
PlacementStatus field is seen only for VM with zone association. Please refer to image below

![image](https://github.com/user-attachments/assets/817c0a0d-c10b-4917-9e48-7adde5d8be78)